lvv/api/lvv/lvv_connection.py

```diff
@@ -72,7 +72,9 @@ def _get_registrations(self, reg_numbers):
             url = f"{self.api_url}registrations/{reg_num}"
             response = self._get(url)
             data = response.json()
-            registrations.append(self._transform(data))
+            registration = self._transform(data)
+            if registration['city'].lower() == 'amsterdam':
```

Comment:
moet weesp en driemond hier ook in?
dan kan je doen:
```suggestion
            if registration['city'].lower() in ['amsterdam', 'weesp', 'driemond']:
```

Reply:
Geldt echt alleen voor Amsterdam.